src/app/zap-templates/templates/chip/helper.js

```diff
@@ -445,6 +446,127 @@ async function chip_endpoint_clusters(options)
   return templateUtil.collectBlocks(clusters, options, this);
 }
 
+/**
+ * If helper that checks if a type is a bitmap which is non-atomic.
```

Comment:
That's not the purpose of this function.  The purpose of this function is to check whether this is a bitmap that is mapped to a BitFlags type, no?

Then in the implementation we can talk about how that correlates with atomic types.

Reply:
I am not sure about Bitflags. Seems like these are the strongly typed enums as well?

Reply:
BitFlags is the strong (i.e. not just an integer) bitmap representation we use.